### PR TITLE
task03_1

### PR DESCRIPTION
**Pull request by natalliakoita**

### CODE REVIEW

**Review by nikonok — APPROVED** [review]

File: lesson_03/main.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	s := words[len(words)-1]
+	return s
+}
+
+// function, that returns the copy of the original slice in reverse order
+func reverse(numbers []int64) []int64 {
+	for i := 0; i < len(numbers)/2; i++ {
+		j := len(numbers) - i - 1
+		numbers[i], numbers[j] = numbers[j], numbers[i]
+	}
```
Comment (denislituev) [inline lesson_03/main.go:44]: Nice!

File: lesson_03/main.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+// function, that prints map values sorted in order of increasing keys
+func printSorted(m map[int]string) {
+	keys := make([]int, len(m))
+	i := 0
+	for k := range m {
+		keys[i] = k
+		i++
+	}
```
Comment (denislituev) [inline lesson_03/main.go:55]: could use **append()** here to populate keys slice with keys

File: lesson_03/main.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	sort.Ints(keys)
+	var keys1 []string
```
Comment (denislituev) [inline lesson_03/main.go:57]: not keys but values

File: lesson_03/main.go [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+	for _, k := range keys {
+		keys1 = append(keys1, m[k])
+	}
```
Comment (denislituev) [inline lesson_03/main.go:60]: there is no reason to create another slice with values only for printing purpose, we can iterate on keys slice and print values.

**Review by denislituev — APPROVED** [review]
Good job! I approve this task, but strongly recomend review comments. It is up to you whether to fix comments or not

**Pull request merged**